html5ever/src/tokenizer/interface.rs

```diff
@@ -27,6 +27,12 @@ pub struct Doctype {
     pub force_quirks: bool,
 }
 
+impl Default for Doctype {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 impl Doctype {
     pub fn new() -> Doctype {
```

Comment:
We should likely remove `Doctype::new()` now and change all references to use the new `Default` implementation.

Reply:
Good call, done.